src/test/ui/consts/constifconst-call-in-const-position.stderr

```diff
@@ -7,12 +7,11 @@ LL | #![feature(const_trait_impl, generic_const_exprs)]
    = note: see issue #76560 <https://github.com/rust-lang/rust/issues/76560> for more information
    = note: `#[warn(incomplete_features)]` on by default
 
-error[E0080]: evaluation of `foo::<()>::{constant#0}` failed
-  --> $DIR/constifconst-call-in-const-position.rs:15:38
+error: ~const can only be applied to `#[const_trait]` traits
+  --> $DIR/constifconst-call-in-const-position.rs:15:17
    |
 LL | const fn foo<T: ~const Tr>() -> [u8; T::a()] {
-   |                                      ^^^^^^ calling non-const function `<() as Tr>::a`
+   |                 ^^^^^^^^^
```

Comment:
Can we make the `Tr` trait as `const_trait`? That should reflect the actual bug here.